scripts/cbmc-viewer/errors.py

```diff
@@ -0,0 +1,139 @@
+"""Format report of property violations found by cbmc."""
+
+import re
+
+ERROR = 0
+WARNING = 1
+PROP = "MISSINGPROP"
+LOOP = "MISSINGLOOP"
+
+
+def assemble_report(result, properties):
+    """Map property violations in result to source line of the violation."""
+    report = {}
+    report[PROP] = {}
+    report[PROP][ERROR] = []
```

Comment:
why are these maps/lists if the set of things you can put into here is so limited?